mpt/core/src/main/java/org/apache/james/mpt/protocol/ProtocolSession.java

```diff
@@ -497,6 +502,24 @@ public boolean isClient() {
         }
     }
 
+    private class ReinitElement implements ProtocolElement {
+
+        private final int sessionNumber;
+
+        public ReinitElement(int sessionNumber) {
+            this.sessionNumber = sessionNumber < 0 ? 0 : sessionNumber;
```

Comment:
I don't think so, all the code is handling such negative sessionNumber... Don't ask me why...